Doc/c-api/buffer.rst

```diff
@@ -42,9 +42,9 @@ Simple objects such as :class:`bytes` and :class:`bytearray` expose their
 underlying buffer in byte-oriented form.  Other forms are possible; for example,
 the elements exposed by an :class:`array.array` can be multi-byte values.
 
-An example consumer of the buffer interface is the :meth:`~io.BufferedIOBase.write`
+An example consumer of the buffer interface is the :meth:`!write`
```

Comment:
Why? This link should work: https://docs.python.org/3/library/io.html#io.BufferedIOBase.write.